confgenerator/loggingmetrics.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Google LLC
```

Comment:
Agent self metrics related logic are mainly in https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/agentmetrics.go. Do we need a separate file (if so, let's rename the other file so it's clearly only doing the metrics subagent side of things), or can we just combine them as one file?

BTW, https://github.com/GoogleCloudPlatform/ops-agent/blob/2ea1dd954bff3d6b5279a3323c4361c88888749f/confgenerator/agentmetrics.go#L22 is designed to work for both logging subagent and metrics subagent's VERSION label. We could potentially re-use it.

Reply:
I think keeping on the same file makes it a bit simpler, so I adopted the suggestion and put the implementation on agentmetrics.go . I think we still need to different pipelines since the VERSION label is different.

Reply:
Thanks!